### ISSUE TITLE
How to make dayjs github site maintainable again

### ISSUE THREAD

**Issue by BePo65** [iamkun/dayjs#1921]
Dayjs is a great library used by many programmers. As a consequence you get a lot of feedback in the issues and pull request sections on github that will require a lot of your time to deal with.

As a consequence the open issues (an pr) pile up pretty fast, when your main job requires all of your time. And then we have the 400 some issues and 80 some pr we have now. When my interpretation of the github insight data is right, you are the only maintainer of this project - what a workload 😃. So perhaps I can apply for a job as 'deputy' maintainer.

What I would recommend to get down to a normal level of open issues and pr is a little multi step action:

1. use github Labels to categorize the open pr. Labels could be the standard one (bug, enhancement etc) and some dayjs only (e.g. locale for pr that define new locales).  
So afterwards it requires less work to scan the list and to identify the important pr that should be merged first.

2. use the github action `Close Stale Issues` to close all pretty old issues after a warning to the contributor.  
During my work on an issue with customParseFormat I scanned all open issues and pr for topics concerning this plugin and that was just a lot of work (I found only 30 issues - many could be closed with a comment - and 8 pr). So this way is impracticable for 400+ issues and I think quite a lot of the issues have a comment that explained what went wrong and how to fix it. These issues could be closed anyway.

3. When the  `Close Stale Issues` did its work, the remaining issues could be tagged with Labels and handled as required.

So I would like to help with these steps, e.g. by adding labels and commenting issues (what I do already at the moment) or by making a first review of open pr. Let me help getting the list of issues and pr down to a normal level. Perhaps there are more people out there that have a little time to spare to help in this process.

**Comment by Bykiev** [comment]
@iamkun, what do you think about it? The first look to the issues (400+) will scare the newbies, as they will think it's a buggy lib, but most of them are just questions.

**Comment by Balastrong** [comment]
With proper moderation most of the issues will get answered/closed and simple bugs fixed, attracting new contributors... we just need some words from @iamkun :)

I would be happy to help as well.

**Comment by BePo65** [comment]
Yeah, when we "got rid of all the old issues" (e..g. with the `Close Stale Issues` github action), proper moderation will be required for the issues.

Today only few issues are closed by the openers, most simply stay open when the question is answered / the handling got explained. In such cases the moderator should IMO actively close the issue.
